tests/framework/installer/ceph_manifests.go

```diff
@@ -1303,6 +1303,8 @@ spec:
           value: "true"
         - name: ROOK_CSI_ENABLE_RBD
           value: "true"
+	- name: ROOK_CSI_ENABLE_GRPC_METRICS
```

Comment:
@leseb is this because of this indentation? if I open the complete file i don't see any issue https://github.com/rook/rook/blob/c3a3dbdf60af0bd0ed7c445ebc05ed2049e755f6/tests/framework/installer/ceph_manifests.go

Reply:
I thought about that too and I had the same reaction while opening the file it seems fine. But I see you updated and github displays it normally so their was something wrong.

Hopefully, that fixes the CI!

Reply:
there was  a mix  of both tab and space, fixed  it now

Reply:
@leseb Thanks for the help :+1: